docs/docs/subscriptions/get-all.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 1
+---
+
+# Get All Subscriptions
+
+Returns a list of all the subscriptions created by your Salable organization.
+
+## Code Sample
+
+```typescript
+import { Salable } from '@salable/node-sdk';
+
+const salable = new Salable('{{API_KEY}}', 'v2');
+
+const subscription = await salable.subscriptions.getAll();
+```
+
+## Parameters
+
+#### options
+
+_Type:_ `GetSubscriptionOptions`
+
+| Option                      | Type             | Description                                                                  |
+| --------------------------- | ---------------- | ---------------------------------------------------------------------------- |
+| status (optional)           | string           | The status of the subscription, e.g. "ACTIVE" "CANCELED"                     |
+| email (optional)            | string           | The email of who purchased the subscription                                  |
+| cursor (optional)           | string           | Cursor value, used for pagination                                            |
+| take (optional)             | string           | The amount of subscriptions to fetch. Default: `20`                          |
+| expand (optional)           | array of strings | Specify which properties to expand. e.g `{ expand: ['product', 'plan'] }`    |
```

Comment:
Type can be `string[]`

Reply:
Also I would add the Required column from other docs here as well instead of placing`(optional)`